docs/network-policy-recommendation.md

```diff
@@ -0,0 +1,128 @@
+# Network Policy Recommendation
+
+## Table of Contents
+
+<!-- toc -->
+- [Introduction](#introduction)
+- [Deployment](#deployment)
+- [Command-line Usage](#command-line-usage)
+  - [Start a policy recommendation job](#start-a-policy-recommendation-job)
+  - [Check the status of a policy recommendation job](#check-the-status-of-a-policy-recommendation-job)
+  - [Get the result of a policy recommendation job](#get-the-result-of-a-policy-recommendation-job)
+<!-- /toc -->
+
+## Introduction
+
+Network policy recommendation is a feature that utilizes the flow records collected by
+[Grafana Flow Collector](network-flow-visibility.md#grafana-flow-collector) and generate
+Kubernetes or [Antrea network policies](https://github.com/antrea-io/antrea/blob/main/docs/antrea-network-policy.md).
```

Comment:
Since `Antrea Network Policies` is a link, I'd suggest to change `Kubernetes` to `Kubernetes Network Policies` for clarity (perhaps also with a link?)